src/app/examples/todos/todos.actions.spec.ts

```diff
@@ -0,0 +1,57 @@
+import * as fromTodos from './todos.reducer';
+
+describe('Todos Actions', () => {
+  describe('TodosAdd', () => {
+    it('should create an action', () => {
+      const action = new fromTodos.ActionTodosAdd({ name: 'test' });
+
+      expect({ ...action }).toEqual({
```

Comment:
Wouldn't it work also with `expect(action).toEqual` ?

Reply:
👋, this fails because you're mixing object types.
You're comparing a `TodosAdd` object with a plain object and this fails with Jasmine, if you would use Jest this would work.